### PR TITLE
Feature/sections form

### PR DESCRIPTION
**Pull request by NicholasLYang**
@jkao1

### CODE REVIEW

File: src/js/modules/forms/components/SectionForm.js [hunk 1]
```diff
@@ -17,20 +25,28 @@ const styles = {
     padding: "5%",
 
   },
+  inputs: {
+    paddingLeft: "2%"
+  },
   titleInput: {
-    padding: "2%"
+    paddingBottom: "2%"
```
Comment (squirrelsquirrel78) [inline src/js/modules/forms/components/SectionForm.js]: These all should be margins, not paddings. Not only is margin more intuitive (margin is outside the element and its border whereas padding is inside), vertical margins auto-collapse. That means that you can do `margin: 2% 0` for all three inputs and the margin won't add up to 4%—it'll be 2%.

Also, why are you using percentages? Just use constants.

File: src/js/modules/sections/selectors.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Created by nicholas on 7/29/17.
+ */
+import { createSelector } from 'reselect'
+
```
Comment (squirrelsquirrel78) [inline src/js/modules/sections/selectors.js:5]: you know how i would name selectors, but this fine ig. just thinkinh out loud.

File: src/js/modules/forms/components/SectionForm.js [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+import Paper                        from 'material-ui/Paper'
+import TextField                    from 'material-ui/TextField'
+import SelectField                  from 'material-ui/SelectField'
+import MenuItem                     from 'material-ui/MenuItem'
+import RaisedButton                 from 'material-ui/RaisedButton'
+import injectSheet                  from 'react-jss'
+import {
+  topLevelSectionsSelector,
+  randomSectionSelector
+}                                   from '../../sections/selectors'
```
Comment (squirrelsquirrel78) [inline src/js/modules/forms/components/SectionForm.js:14]: you really should import the sections module, then get selectors

**Review by squirrelsquirrel78 — CHANGES_REQUESTED** [review]
Small changes here and there

File: src/js/modules/forms/components/SectionForm.js (hunk 1 above)
Comment (NicholasLYang) [reply src/js/modules/forms/components/SectionForm.js]: Changed to margin. I'm using percents for now. If they become a problem, I'll switch to constants

**Review by squirrelsquirrel78 — APPROVED** [review]

**Pull request merged**